_locales/en/messages.json

```diff
@@ -646,6 +646,12 @@
     "configurationBatteryScale": {
         "message": "Voltage Scale"
     },
+    "configurationVbatPidCompensation": {
+        "message": "Vbat PID Compensation"
+    },
+    "configurationVbatPidCompensationHelp": {
+        "message": "Increase the PID values to compensate when Vbat gets lower. This will give more constant flight characteristics throughout the flight."
```

Comment:
I don't want to be the spoilsport, but I also want to make sure I am not seeing things. The only reference to `calculateBatteryPercentage` that I could find is in the display code:

```
~/git/betaflight$ find . -name .git -o -name obj -prune -o -type f -exec grep calculateBatteryPercentage {} \; -print
uint8_t calculateBatteryPercentage(void)
./src/main/sensors/battery.c
uint8_t calculateBatteryPercentage(void);
./src/main/sensors/battery.h
        uint8_t batteryPercentage = calculateBatteryPercentage();
./src/main/io/display.c
```